src/v/datalake/datalake_manager.cc

```diff
@@ -86,6 +86,8 @@ make_record_translator(const model::iceberg_mode& mode) {
 
 class disk_manager_impl : public translation::scheduling::disk_manager {
 public:
+    static constexpr ss::shard_id shard = 0;
```

Comment:
nit: maybe manager_shard or something? read inline, it's easy to conflate this with `from`

Reply:
fixed